bot/cogs/wolfram.py

```diff
@@ -0,0 +1,68 @@
+import discord
+from discord.ext import commands
+from urllib import parse
+from io import BytesIO
+import aiohttp
+import os
+
+
+APPID = os.environ.get("WOLFRAM_APPID")
```

Comment:
any env variables/constants should be in `settings.py`.

Reply:
by this do you mean that i define `APPID` in settings.py, and then import it on the cog?